sdk/src/main/scala/com/horizen/account/api/http/AccountEthRpcRoute.scala

```diff
@@ -121,4 +128,17 @@ case class AccountEthRpcRoute(
       }
     }
   }
+
+  private def getClientVersion: String = {
+    val default = "unknown"
+    val architecture = Try(System.getProperty("os.arch")).getOrElse(default)
+    val javaVersion = Try(System.getProperty("java.specification.version")).getOrElse(default)
+    val pom = Try(new MavenXpp3Reader().read(new FileReader(new File("pom.xml")))).toOption
```

Comment:
Not sure if we can rely on using this with a relative path.

Reply:
You could try `this.getClass.getPackage.getImplementationVersion`.

Note: This will return `null` in a non-packaged environment, i.e. during development when run from the IDE. It requires the JAR manifest and should work in a properly packaged build. Maybe also use the default of "unknown" or "development" when it returns `null`.

Reply:
I have a doubts about this implementation as well.
We need something more stable, like here: https://stackoverflow.com/a/70307580